### ISSUE TITLE
Map credits being overlayed on Insert Image popup that is failing

### ISSUE THREAD

**Issue by waridrox** [publiclab/plots2#9442]
### Please describe the problem (or idea)

**What happened just before the problem occurred? Or what problem could this idea solve?**
Map credits like **Leaflet, Mapbox and OpenStreetMap** show up when the insert image popup is displayed. The image insert functionality is also not working from device / from URL

<img width="622" alt="Screenshot 2021-04-04 at 2 30 03 AM" src="https://user-images.githubusercontent.com/58583793/113500883-39322d00-953f-11eb-8db8-012c56f38acc.png">

### Please show us where to look

https://publiclab.org/

### What's your PublicLab.org username?

waridrox

### Browser, version, and operating system

Chrome v89

**Comment by Priyaraj17** [comment]
May I work on this issue?

**Comment by waridrox** [comment]
@Priyaraj17 Yes, go for it 🚀 maybe tweaking with the z-index property of the credits can resolve this...

**Comment by Priyaraj17** [comment]
I tried to reproduce the error, but things are fine on my PC. I am using Linux. Here is a screenshot:
![image](https://user-images.githubusercontent.com/40101776/113547341-53394180-960b-11eb-9ed0-e2610a83eac0.png)
Can you please take a look?

**Comment by waridrox** [comment]
The bug is kind of tricky to produce, you have to give enough room for the image upload prompt to take over the map preview like this - 


https://user-images.githubusercontent.com/58583793/113548884-fc813700-960d-11eb-8d6a-5187e201a42e.mp4

**Comment by Priyaraj17** [comment]
Okay, Got it, Thanks
